18760.yml

```diff
@@ -20,29 +20,21 @@ jobs:
         with:
           script: |
             const submission = require('./submission.json')
-            const herokuURL =  submission.checklist[0].result
-            if ((herokuURL.split("/").length >= 2) && herokuURL.includes("herokuapp.com")) {
-              return herokuURL
-            } else {
-              throw "The submitted URL seems to be invalid"
-            }
+            return submission.checklist[0].result
           result-encoding: string
       - name: Report invalid repository URL in submission
         if: steps.get-submission-url.outcome != 'success'
         uses: pupilfirst/actions/grading@v1
         with:
           fail_submission: true
-          feedback: The submitted URL is either unavailable or not a valid Heroku app. Please make sure that you submit a valid Heroku application URL.
+          feedback: We failed to extract the URL for your live web application from your submission. Please submit again; if this issue persists, please reach out to our team for support.
```

Comment:
This won't be required anymore as the `URL` as `get-submission-url` is no longer throwing an error.